### DISCUSSION TITLE
Release 5.4.0 & GitHub Actions

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1554] [Team Posts]
Hello guys,

I would like to release 5.4.0 with some stuff we willing to package in it.
I ditched some of them. I don't know if we should put all PR from the milestone but I think we have enough stuff to push a release.
I'm waiting for some of them from their owner.

Also, I would like to ditch Travis and use GitHub Actions for tests. Travis has been sooo slow recently and GitHub Actions are pretty fast to be triggered.
What do you think?

**Comment by j0k3r** [comment]
The PR is ready (https://github.com/serverless-heaven/serverless-webpack/pull/658) to ditch both Travis & AppVeyor because GitHub Actions can handle windows server too.
Let me know what do you think and then @HyperBrain will need to update requirement for PR to be merged.

**Comment by miguel-a-calles-mba** [comment]
I really like GitHub Actions. We are using it to publish the npm package too.

**Comment by HyperBrain** [comment]
Hi Jérémy,
just let me know what I should do exactly :)

**Comment by j0k3r** [comment]
Hi Frank,

You should update the required status checks in the branch protection rules here: https://github.com/serverless-heaven/serverless-webpack/settings/branches

Untick appveyor & travis and tick `Linter` & all 4 `Tests / Node.js`. If you don't know what to tick, share a screenshot of the list.
And it should be fine :)

Thanks!

**Comment by HyperBrain** [comment]
Hmmm.... I only see AppVeyor and Travis but no Linter & Tests

**Comment by HyperBrain** [comment]
[image: image.png]

**Comment by miguel-a-calles-mba** [comment]
The Linter and Tests will show up after the GitHub Actions runs for the first time from recollection.

I believe unchecking AppVeyor and Travis is needed for now. Requiring the Linter and Tests should be done after the PR merges and those options appear.

**Comment by j0k3r** [comment]
As @miguel-a-calles-mba says, we should first disabled Travis and AppVeyor, then merge my PR and new checks will show up in the branch settings

**Comment by HyperBrain** [comment]
Ok. will do it this evening and ping here

**Comment by j0k3r** [comment]
Let me know when you'll do it @HyperBrain. Travis is beginning to be awfully slow these days 😫

**Comment by j0k3r** [comment]
Hi @HyperBrain hope everything is fine.
Happy new year.
Could you tell me when you'll be available to make the switch? Or maybe give someone ability to do it.
Thanks

**Comment by HyperBrain** [comment]
AppVeyor and Travis are now unticked. Let me know when the new ones will be visible

**Comment by j0k3r** [comment]
Great! Thanks. New ones should be in place now as @miguel-a-calles-mba merged the PR

**Comment by miguel-a-calles-mba** [comment]
Whoops. I wrote the message below yesterday, but forgot to hit send.

@HyperBrain , the PR merged. You should now see "Tests" and "Linter" checks in the settings. They might have a GitHub logo.

**Comment by HyperBrain** [comment]
Will check as soon as I am at home

**Comment by HyperBrain** [comment]
All ticked now

**Comment by HyperBrain** [comment]
... including coverage upload
